ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/resources/streaming/TopicStream.java

```diff
@@ -64,7 +64,7 @@ public RecordFormatter(SchemaRegistryClient schemaRegistryClient, String topicNa
       return StreamSupport
           .stream(records.records(topicName).spliterator(), false)
           .map((record) -> {
-            if (record == null) {
+            if (record == null || record.value() == null) {
```

Comment:
is there actually a case where record would be null?

Reply:
What do you mean? `null` values are perfectly viable in any kafka record. This batch of `ConsumerRecords` is coming directly from the consumer and there is no filtering done on them to weed out nulls before this.

Reply:
no, I meant do we still need the `record == null` check, since that's what was wrong in the first place? Would poll ever return a null record (not value)?

Reply:
I don't think we strictly need it, because I don't think the consumer can return null records. But it is better to be safe in this case IMO.